README.md

```diff
@@ -355,6 +355,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 
 * [Better Reflection](https://github.com/Roave/BetterReflection) - AST-based reflection library that allows analysis and manipulation of code
 * [Code Climate](https://codeclimate.com) - An automated code review.
+* [Editorconfig-Checker](https://github.com/editorconfig-checker/editorconfig-checker.php) - A command line utility which verifies if your files implement your `.editorconfig` rules.
```

Comment:
if => that

Reply:
Also, maybe use https://github.com/editorconfig-checker as link?

Reply:
Fixed.